app/boards/intel_adsp_ace15_mtpm.conf

```diff
@@ -1,17 +1,62 @@
 CONFIG_METEORLAKE=y
+CONFIG_IPC_MAJOR_4=y
+
+# turn off SOF drivers
 CONFIG_INTEL_DMIC=n
 CONFIG_INTEL_DMIC_NHLT=n
 CONFIG_INTEL_SSP=n
 CONFIG_INTEL_ALH=n
-CONFIG_RIMAGE_SIGNING_SCHEMA="mtl"
-CONFIG_IPC_MAJOR_4=y
-CONFIG_INTEL_ADSP_IPC=y
-CONFIG_DW_ICTL_ACE_V1X=y
 CONFIG_COMP_SRC=y
-CONFIG_INTEL_ADSP_TIMER=y
+
+CONFIG_COMP_SRC_IPC4_FULL_MATRIX=y
+
+# enable Zephyr drivers
+CONFIG_ZEPHYR_NATIVE_DRIVERS=y
+CONFIG_DAI=y
+CONFIG_DAI_INIT_PRIORITY=70
+CONFIG_DAI_INTEL_DMIC=y
+CONFIG_DAI_DMIC_HW_IOCLK=38400000
+CONFIG_DAI_INTEL_DMIC_NHLT=y
+CONFIG_DAI_DMIC_HAS_OWNERSHIP=y
+CONFIG_DAI_DMIC_HAS_MULTIPLE_LINE_SYNC=y
+CONFIG_DAI_INTEL_SSP=y
 CONFIG_DMA=y
+CONFIG_DMA_INTEL_ADSP_GPDMA=y
+CONFIG_DMA_DW_LLI_POOL_SIZE=50
+CONFIG_INTEL_ADSP_TIMER=y
+
+CONFIG_HEAP_MEM_POOL_SIZE=8192
+CONFIG_RIMAGE_SIGNING_SCHEMA="mtl"
+
 CONFIG_FORMAT_CONVERT_HIFI3=n
+CONFIG_PCM_CONVERTER_FORMAT_S16_C16_AND_S16_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S32_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S24_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S24_C24_AND_S24_C32=y
+CONFIG_PCM_CONVERTER_FORMAT_S24_C32_AND_S24_C24=y
+CONFIG_PCM_CONVERTER_FORMAT_S16_C32_AND_S16_C32=y
+CONFIG_LOG=y
+CONFIG_LOG_MODE_DEFERRED=y
+CONFIG_LOG_BACKEND_ADSP=y
```

Comment:
CONFIG_LOG_BACKEND_ADSP is not needed too, we can leave only mtrace enabled by default